core/schema/engine.go

```diff
@@ -17,7 +17,7 @@ var _ SchemaResolvers = &engineSchema{}
 
 func (s *engineSchema) Install() {
 	dagql.Fields[*core.Query]{
-		dagql.Func("daggerEngine", s.engine).
+		dagql.Func("engine", s.engine).
```

Comment:
Note: this is a breaking change.

Can we create a new method and deprecate the old one? I don't think anyone *is* depending on it yet, but it's good to avoid breaking things in patch releases wherever possible.

Reply:
sure, SGTM. Adding a new method seems ok.

Reply:
Make sure to deprecate the old one 😄

Reply:
yep, will mark the old one as deprecated :+1:

Reply:
As mentioned below - this *isn't* a breaking change for modules - which is what we care about.

The API still changes, so `dagger core dagger-engine` becomes `dagger core engine`, and external SDKs are still breaking, but modules don't have access to this type *in the first place*. So I think we can take this whenever.